python-doctest/options1.txt

```diff
@@ -0,0 +1,10 @@
+>>> 5 < 7  # doctest: +DONT_ACCEPT_TRUE_FOR_1
```

Comment:
Maybe it makes sense to call this file something more descriptive, like "options_with_directives.txt" or something like that, likewise, the previous options.txt, could be called "options_no_directives.txt".